lib/stream/leo-stream.js

```diff
@@ -849,13 +849,13 @@ module.exports = function(configure) {
 				enableLogging: true,
 				snapshot: opts.snapshot
 			}, opts.chunk || {});
-			chunkOpts.gzip = !opts.firehose;
+			chunkOpts.gzip = opts.gzipFirehose || !opts.firehose;
```

Comment:
@czirker Or is that what this does? The chunker does the gzipping?

Reply:
Correct.  The chunker will do the zipping.  The user just needs to specify that they want to enable the feature using `gzipFirehose:true`